ui/imports/shared/views/PasswordView.qml

```diff
@@ -153,14 +153,23 @@ ColumnLayout {
         color: Theme.palette.directColor1
     }
 
-    StatusBaseText {
-        id: introTxtField
-        Layout.fillWidth: true
-        text: "%1 <font color=\"%2\">%3</font>".arg(root.introText).arg(Theme.palette.dangerColor1).arg(root.recoverText)
-        font.pixelSize: root.highSizeIntro ? 15 : 12
-        color: Theme.palette.baseColor1
-        wrapMode: Text.WordWrap
-        horizontalAlignment: TextEdit.AlignHCenter
+    ColumnLayout {
+        Layout.alignment: Qt.AlignHCenter
+        spacing: 4
+
+        StatusBaseText {
+            Layout.alignment: Qt.AlignHCenter
+            text: qsTr("Create a password to unlock Status on this device & sign transactions.")
```

Comment:
```suggestion
            text: root.introText
```

Reply:
Thanks for the comment, that's right, making an update.